spec/model/car_spec.rb

```diff
@@ -0,0 +1,11 @@
+require_relative '../../model/car'
+
+describe Car do
+  let(:car) { Car.new("CJ30PXP", "2020-11-24") }
+  
+  it "stores an id and a date" do
```

Comment:
same remark here regarding double quotes. You can write all the strings from this class by using single quotes